monkey/monkey_island/cc/models/attack_mitigation.py

```diff
@@ -0,0 +1,23 @@
+from mongoengine import Document, StringField, DoesNotExist
+
+
+class AttackMitigation(Document):
+
+    technique_id = StringField(required=True, primary_key=True)
+    name = StringField(required=True)
+    description = StringField(required=True)
+
+    @staticmethod
+    def get_mitigation_by_technique_id(technique_id: str) -> Document:
+        try:
+            return AttackMitigation.objects.get(technique_id=technique_id)
+        except DoesNotExist:
+            raise Exception("Attack technique with id {} does not exist!".format(technique_id))
+
+    @staticmethod
+    def add_mitigation_from_stix2(mitigation_stix2_data):
+        mitigation_model = AttackMitigation(technique_id=mitigation_stix2_data['external_references'][0]['external_id'],
```

Comment:
If I understand correctly `[0]` is because that's where the MITRE reference is 
I'd prefer if we found the one in the list where source-name == mitre-attack